{"name": ["shgo_slsqp_sobol_cube", "shgo_powell_sobol_cube", "shgo_nelder_sobol_cube", "scipy_slsqp_cube", "scipy_powell_cube", "scipy_nelder_cube", "scipy_lbfgsb_cube", "pysot_ei_cube", "pysot_lcb_cube", "pysot_random_cube", "pysot_srbf_cube", "pysot_dycors_cube", "ax_default_cube", "optuna_cmaes_cube", "optuna_tpe_cube", "optuna_random_cube", "optuna_random_cube_clone", "optuna_random_cube_clone_1", "optuna_random_cube_clone_2", "platypus_genetic_cube", "platypus_evolutionary_cube", "platypus_nsgaii_cube", "platypus_cmaes_cube", "platypus_gde3_cube", "platypus_ibea_cube", "platypus_moead_cube", "platypus_omopso_cube", "platypus_smpso_cube", "platypus_spea2_cube", "platypus_epsmoea_cube", "nevergrad_ngopt_cube", "nevergrad_ngopt4_cube", "nevergrad_de_cube", "nevergrad_portfolio_cube", "nevergrad_oneplus_cube", "nevergrad_hammersley_cube", "nevergrad_ngopt8_cube", "hyperopt_tpe_cube", "hyperopt_atpe_cube", "hyperopt_rand_cube", "pymoo_nelder_cube", "pymoo_nsga3_cube", "pymoo_unsga3_cube", "pymoo_pattern_cube", "pymoo_nsga2_cube", "pymoo_unsga3_cube", "skopt_gp_default_cube", "ultraopt_random_cube", "ultraopt_forest_cube", "ultraopt_gbrt_cube", "ultraopt_etpe_cube", "bayesopt_ucb_cube", "bayesopt_ei_cube", "bayesopt_poi_cube", "dlib_cube", "dlib_default_cube", "nlopt_direct_cube", "nlopt_ags_cube", "nlopt_esch_cube", "nlopt_isres_cube", "nlopt_crs2lm_cube", "nlopt_directr_cube", "nlopt_directo_cube", "nlopt_directl_cube", "bobyqa_default_cube", "bobyqa_noise_cube", "hebo_cube_sequential", "hebo_cube_batch", "hebo_sequential_cube", "hebo_batch_cube"], "count": [17, 13, 21, 54, 18, 0, 38, 70, 51, 63, 58, 68, 36, 58, 62, 58, 63, 61, 48, 52, 76, 61, 58, 61, 47, 71, 50, 62, 57, 49, 49, 48, 55, 72, 56, 46, 51, 46, 64, 64, 56, 62, 117, 60, 66, 0, 57, 55, 59, 61, 72, 59, 56, 65, 55, 62, 62, 46, 52, 49, 60, 48, 63, 75, 55, 45, 39, 50, 10, 16], "rating": [1805.9779171859366, 1750.4829965284948, 1870.7409529168167, 1040.473354525541, 1393.9159924140356, 1600, 1129.9732162016198, 1946.261733717565, 1772.3227092952877, 1207.6988418819567, 1943.0121379697011, 2012.5908660196326, 1849.27893026336, 1833.891927568884, 1664.5883901016282, 1600, 1600, 1600, 1600, 1299.214825029238, 1223.507137398093, 1312.2505638779037, 1359.7715679172436, 1330.8765774965223, 1312.4077617583725, 1467.1178294492443, 1260.4954741626898, 1374.9462586429127, 1324.3001279563452, 1256.8744038358052, 1826.0309602750917, 1813.6671606781842, 1428.523896137495, 1451.9809754946752, 1618.3441840042083, 1432.7576290683282, 1819.0003090630134, 1416.9879259659147, 1439.1186665532684, 1567.5686894948226, 1881.3654090707766, 1524.74236069504, 1557.5276467077097, 1771.9566098460402, 1452.0567706668114, 1600, 1971.737009627781, 1265.03630101545, 1746.9608902712819, 1853.0827235125844, 1900.1394582180856, 1867.9453939818534, 1917.7574466911653, 2040.9986753306923, 1895.5961452616398, 1869.588136887373, 1928.1467004159572, 1779.2857651034199, 1102.7764684791928, 1382.3680790422134, 1743.4966683107596, 1922.8705883935988, 1791.0883214686964, 2018.3090207359169, 1884.8122774853985, 1908.9313675584472, 1840.6667637378803, 1843.5893493373774, 1673.0036096453116, 1620.1051690146335], "traceback": ["passing", "passing", "passing", "passing", "passing", "not yet run", "Black took 113 function evals when instructed to use 135", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 76, in optimizer_game\n    black_best_val, black_best_x, black_feval_count = black(objective, n_trials=n_black_trials,\n  File \"/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/humpday/optimizers/platypuscube.py\", line 108, in platypus_epsmoea_cube\n    return platypus_cube(objective=objective, n_trials=n_trials, n_dim=n_dim, with_count=with_count,\n  File \"/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/humpday/optimizers/platypuscube.py\", line 51, in platypus_cube\n    best_obj, best_x = feasible_solution_obj[0]\nIndexError: list index out of range\n", "passing", "Black took 132 function evals when instructed to use 131", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "not yet run", "passing", "passing", "passing", "Black took 131 function evals when instructed to use 131", "Black took 134 function evals when instructed to use 134", "passing", "passing", "passing", "passing", "passing", "passing", "Black took 131 function evals when instructed to use 131", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 119, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 41, in optimizer_game\n    white_best_val, white_best_x, white_feval_count = white(objective, n_trials=n_white_trials, n_dim=n_dim,\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 52, in hebo_sequential_cube\n    return hebo_cube_factory(objective=objective, n_trials=n_trials, n_dim=n_dim, with_count=with_count, n_suggestions=1)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 39, in hebo_cube_factory\n    rec = opt.suggest(n_suggestions=n_suggestions) # <-- don't change this\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/optimizers/hebo.py\", line 124, in suggest\n    rec = opt.optimize(initial_suggest = best_x, fix_input = fix_input).drop_duplicates()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 125, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 119, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 76, in optimizer_game\n    black_best_val, black_best_x, black_feval_count = black(objective, n_trials=n_black_trials,\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 56, in hebo_batch_cube\n    return hebo_cube_factory(objective=objective, n_trials=n_trials, n_dim=n_dim, with_count=with_count, n_suggestions=10)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 39, in hebo_cube_factory\n    rec = opt.suggest(n_suggestions=n_suggestions) # <-- don't change this\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/optimizers/hebo.py\", line 124, in suggest\n    rec = opt.optimize(initial_suggest = best_x, fix_input = fix_input).drop_duplicates()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 125, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n"], "active": [true, true, true, true, true, false, true, true, true, true, true, true, false, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, false, false, false, false]}